itamae.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "thor"
-  spec.add_runtime_dependency "specinfra", [">= 2.37.0", "< 3.0.0"]
+  spec.add_runtime_dependency "specinfra", [">= 2.43.11", "< 3.0.0"]
```

Comment:
2.43.11 is included code that call Docker::Container#exec with some options.
#=> https://github.com/mizzy/specinfra/pull/487/files

So, I can call Itamae::Backend::Docker#run_command with `{ tty: true }`.
#=> https://github.com/itamae-kitchen/itamae/pull/168/files#diff-d02df415fdc33c86a44dc45454709c68R252